src/index.js

```diff
@@ -9,6 +9,8 @@ if (!spy) throw new Error("mobx-react requires mobx to be available")
 if (typeof rdBatched === "function") configure({ reactionScheduler: rdBatched })
 else if (typeof rnBatched === "function") configure({ reactionScheduler: rnBatched })
 
+// TODO: re-export mobx-react-lite stuff?
```

Comment:
I think it should be safe enough as everything is prefixed with `use` so it should generally understood as a hook to be used in FC only.